### PR TITLE
DSYS-73: Narrow sidebar js

### PR DESCRIPTION
**Pull request by lucianoratamero**
**Link to task:** https://labcodes.atlassian.net/browse/DSYS-73

**Details:**
The components that this PR contains are intentionally different from the specs, since it would take a while to refactor them. 
We focused on testing and functionality instead.
There are some small details missing and needed to be refactored, but they're already considered on the [refactor task](https://labcodes.atlassian.net/browse/DSYS-123).

### CODE REVIEW

File: stories/NarrowSidebarVivid.stories.mdx [hunk 1]
```diff
@@ -28,104 +20,104 @@ import Icon from '../labsystem/src/Icon';
...
-            <FooterButton icon="logout" label="Logout" />
-          </Footer>
+      <div className="lab-narrow-sidebar--stories lab-narrow-sidebar--stories-vivid-coral">
+        <NarrowSidebar withDividers isVivid >
+          <NarrowSidebar.Header >
+            <NarrowSidebar.Logotype logoSrc="./favicon.ico" altText="User photo" />
+            <NarrowSidebar.CollapseButton />
+          </NarrowSidebar.Header>
+          <NarrowSidebar.Body  >
+            <NarrowSidebar.Item onClick={() => {}} icon="calendar" label="Teste" />
```
Comment (olgapinheiro) [inline stories/NarrowSidebarVivid.stories.mdx:105]: missing `isActive` in the `coral` example

File: playgrounds/NarrowSidebarPlayground.js [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+              type={showBodyConfigs ? "collapse-open" : "collapse-closed"}
+            />
+          </p>
+          {showBodyConfigs ? (
+            <React.Fragment>
+              <span className="lab-playground__item">
+                <TextInput
+                  id="itemIcon"
+                  label="itemIcon"
+                  value={itemIcon}
```
Comment (olgapinheiro) [inline playgrounds/NarrowSidebarPlayground.js:295]: What do you think of using a Select input to choose the `itemIcon` and the `footerButtonIcon`?
I was going to do that, but didn't have time

File: playgrounds/NarrowSidebarPlayground.js [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+              <span className="lab-playground__item">
+                <TextInput
+                  id="avatarCaption"
+                  label="avatarCaption"
+                  value={avatarCaption}
+                  onChange={this.handlePropChangeText}
+                />
+              </span>
+            </React.Fragment>
+          ) : null}
```
Comment (olgapinheiro) [inline playgrounds/NarrowSidebarPlayground.js:360]: Maybe it would be good to add some padding between the avatar props and the `footerButton` props?

**Review by olgapinheiro — CHANGES_REQUESTED** [review]
Hey, seems to be working fine!
I added a few small comments. Most of them are totally optional, except for the one about the coral vivid example.

File: playgrounds/NarrowSidebarPlayground.js (hunk 2 above)
Comment (lucianoratamero) [reply playgrounds/NarrowSidebarPlayground.js:318]: because of the due date, I'll not work much on more playground improvements, sorry about that =/

**Review by debscorreia — APPROVED** [review]

**Pull request merged**